shared/tracker/render.js.flow

```diff
@@ -2,13 +2,20 @@
 
 import {Component} from 'react'
 
-import type {Props as BioProps} from '../common-adapters/user-bio'
+import type {UserInfo} from '../common-adapters/user-bio'
 import type {Props as ProofsProps} from '../common-adapters/user-proofs'
 import type {ActionProps} from './action.render'
 import type {HeaderProps} from './header.render'
 
 export type RenderPropsUnshaped = {
-  bioProps: BioProps,
+  bioProps: {
+    // FIXME: this is the UserBio props minus the avatar, which doesn't make
+    // sense to pass from the smart component. We should remove this after a
+    // refactor of the tracker UI.
```

Comment:
It was necessary to copy the `BioProps` type information here due to the hacky way the tracker components are laid out. Using `BioProps` would require us to pass the avatar size in the smart component and the same value would apply both the desktop and native components.